From a745837e936b0c996a348329db1a985b2543bf87 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Tue, 18 Jun 2024 09:42:41 +0530
Subject: [PATCH] [WA] Fixed build error in bionic.

Observed build issue due to unavaiable reference of __sF[].
For now removing flag to make it avaiable.

Tests: Build the EB. there was no issue due to this change.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 libc/include/stdio.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libc/include/stdio.h b/libc/include/stdio.h
index 279c6584a..50a016b33 100644
--- a/libc/include/stdio.h
+++ b/libc/include/stdio.h
@@ -69,7 +69,7 @@ extern FILE* _Nonnull stderr __INTRODUCED_IN(23);
 #define stderr stderr
 #else
 /* Before M the actual symbols for stdin and friends had different names. */
-extern FILE __sF[] __REMOVED_IN(23, "Use stdin/stdout/stderr");
+extern FILE __sF[];// __REMOVED_IN(23, "Use stdin/stdout/stderr");
 
 #define stdin (&__sF[0])
 #define stdout (&__sF[1])
-- 
2.34.1

